packages/flutter/lib/src/rendering/object.dart

```diff
@@ -2249,20 +2248,20 @@ abstract class RenderObject with DiagnosticableTreeMixin implements HitTestTarge
   static bool debugCheckingIntrinsics = false;
 
   bool _debugRelayoutBoundaryAlreadyMarkedNeedsLayout() {
-    if (_relayoutBoundary == null) {
+    if (_isRelayoutBoundary == null) {
```

Comment:
Not related to this PR, but once you change `_isRelayoutBoundary` from `bool?` to `bool`, is it still possible to keep this assert? Since here it actually cares whether it is null, instead of treating `null` as false like everywhere else?